src/others/sysmon/lv_sysmon.c

```diff
@@ -131,8 +133,11 @@ static void perf_monitor_disp_event_cb(lv_event_t * e)
             info->refr_start = lv_tick_get();
             break;
         case LV_EVENT_REFR_FINISH:
-            info->refr_elaps_sum += lv_tick_elaps(info->refr_start);
-            info->refr_cnt++;
+            uint32_t elapsed_time = lv_tick_elaps(info->refr_start);
+            if (elapsed_time > 0) {
```

Comment:
What I did here was a quick and dirty to remove the counts from being made for frames that nothing is being rendered. It was only for testing purposes and nothing more.

Reply:
I think what I did was I added the omitting anything that was a zero time before I changed the code the handles the validation of work being performed and I forgot to remove the checking for a zero. I didn't have a blinking carat in my test and I was doing a double buffer full frame buffers for an 800 x 600 x 32 display.  which is quite a bit of work to do so it would take longer than 1 millisecond so the displayed FPS would not have been skewed at all by that being there.